packages/controls/src/widget_string.ts

```diff
@@ -274,6 +274,10 @@ class TextView extends DescriptionView {
 
         this.textbox = document.createElement('input');
         this.textbox.setAttribute('type', this.inputType);
+        if (this.model.get('description').length === 0) {
```

Comment:
This check should probably be moved into update_title, and update_title called when the description changes too. That way if someone changes the description text, the appropriate thing is done with the title attribute.